APIs/SweccAPI.py

```diff
@@ -22,3 +22,21 @@ def auth(self, discord_username, id, username):
 
         response = requests.put(f"{self.url}/members/verify-discord/", headers=headers, json=data)
         return response.status_code
+
+    def leetcode_leaderboard(self, order_by=None):
+        logging.info("Fetching leetcode leaderboard order by %s", order_by)
+        headers = {
+            "Authorization": f"Api-Key {self.api_key}",
+            "Content-Type": "application/json",
+        }
+
+        params = {"order_by": order_by} if order_by else {}
```

Comment:
Should we just do the default ordering is most solved to least overall? Instead of no ordering?

Reply:
that is default but I'll add to make more explicit

(from swecc-server)
```py
order_by = self.request.query_params.get("order_by", "total")
time_range = self.request.query_params.get("updated_within", None)
```